presto-main/src/main/java/com/facebook/presto/sql/analyzer/SemanticErrorCode.java

```diff
@@ -83,4 +83,6 @@
     AMBIGUOUS_FUNCTION_CALL,
 
     INVALID_PARAMETER_USAGE,
+
+    VIEW_CANNOT_BE_RECURSIVE,
```

Comment:
Move this above with the `VIEW_` errors (there's no strict rule for how these error codes are organized, but it's good to keep related things together)